proposals/PROPOSALS.md

```diff
@@ -1,141 +1,127 @@
 
-- A contributor needs computational resources for CI, or to produce stable Hackage subsets
-- A contributor wants to write educational materials covering a yet-unwritten-about portion of Haskell, but requires an editor
-- An open source project requires funding for volunteers engaged in Summer of Code-type work on core tooling for Haskell
+## Decisions
 
-In each of these cases, the author of the associated HFTP should detail the resource request for machines, an editor, or in terms of volunteer funding needed to meet their delivery goals. Throughout the HFTP process, these resource requests will be considered and weighed against the existing demand for HF resources. If accepted, these resource requests will be honored by the HF.
+The committee will make all decisions using a simple majority without secret ballots.
+Discussions and vote tallies regarding potential committee members are to be kept strictly confidential, while other discussions and votes should be transparent to the public.
+Decisions should be made with the participation of 2/3 of the current members of the committee, rounded up.
+The committee may decide to hold regular meetings, or to collaborate asynchronously, or some mix of the two.
+For instance, a valid decision can be made by having a non-quorum meeting discuss a matter, summarize the results for the other members, and then vote by email based on the summary.
+Disputes will be resolved by the ED.
 
-### Proposal Size
+## Process
 
-There is no size floor or ceiling for project submissions. Each HFTP will be weighed against the existing resource pool to decide whether a proposal is ready for acceptance. In particular, small quality of life proposals are as welcome as epic community-shifting proposals - we do not judge. However, this is why step 1 (informal discussion) is so important: often, it may be faster and easier to just make that Pull Request and find people who are free to do it with you! If, for some reason, the discovery and discussion phase does not catch these issues, it will be raised by HFTT members when appropriate. When a proposal, big or small, is inappropriate for the venue, we will be sure to make it known.
+The TWG, as an advisory committee to the Haskell Foundation and to the community at large, exists primarily to aid in making good decisions. 
 
-## What's the process for submitting a HFTP?
+Proposals should explicitly state which **category** they are in (RFC, Community Project, HF Project), or describe why they do not fit into an existing category but should be considered by the committee anyway.
+Additionally, all proposals should address the following concerns:
+ * **Problem statement:** What problem is the proposed work intended to solve? What are the requirements against which a solution should be evaluated? How will solving this problem benefit the community?
+ * **Prior art:** What other similar work has been done in the community, and how is the proposal related to it?
```

Comment:
Suppose someone wants to write a proposal but is not well informed as to prior art. Is it acceptable to write a proposal and explicitly ask for comments directing them to prior art? Prior art discovery is hard, and we have had trouble in the past when folks have posted an idea only to have community members say that the problem already has a solution, so explicit guidelines here may be very helpful.

Reply:
People should do some due diligence search for prior art before a proposal -- that's precisely what the pre-proposal is for, among other things! They may of course miss stuff -- that's fine. But asking that they address this to the extent they know in a proposal also sounds fine.

Reply:
Yeah, I think it's reasonable to evaluate proposals based on this concern, and to help people who are working in good faith but have missed a few things to find the things they missed. I do think that proposers should do their homework, but also that we should help them with it.

Reply:
I added a little bit after this list:
> Perfection is not necessary. For instance, a missing piece of prior art, a technical plan that is missing an important step, or an overlooked group of stakeholders will not doom a proposal, and the committee will suggest improvements and additions as needed. All that's required is a good faith attempt to address these concerns in the first draft.

Does this address your concerns, @goldfirere and @gbaz?

Reply:
I'm satisfied, yes. Thanks. Leaving this to @gbaz to resolve if he is similarly happy.